transformer_xl/data_loader.py

```diff
@@ -0,0 +1,98 @@
+""" Stolen from https://github.com/nshepperd/gpt-2/blob/finetuning/src/load_dataset.py"""
```

Comment:
Odd to use the GPT-2 tokenizer on TXL, any reason for this?

Reply:
Ah, now I see it's copy-pasta and you're passing in a different encoder. Can you either remove the __main__ below or update it? Also probably want to change the hash so it doesn't collide with cached files from GPT-2 encoding.

Reply:
Oops, that’s a typo— the script itself has been modified to support the transformer-xl tokenizer. When loaded into train.py the script runs fine; I will fix up main() and re-submit.

Reply:
^^ Yep, I’ll change the hash as well